modules/azure/resourcegroup.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"testing"
 
+
```

Comment:
Ah this extra whitespace is causing our go format check to fail. Can you remove this?